connectors/embeddings-vector-database/pom.xml

```diff
@@ -33,16 +33,11 @@
       <groupId>dev.langchain4j</groupId>
       <artifactId>langchain4j</artifactId>
     </dependency>
-    
-    <dependency>
-      <groupId>dev.langchain4j</groupId>
-      <artifactId>langchain4j-ollama</artifactId>
-    </dependency>
```

Comment:
Good catch! Can we also remove the PGVector dependency? We had both Ollama + PGVector support in the beginning, but then decided to remove it for the moment, so probably that library is also a leftover.